Numerical_solution_of_SEIRD.Rmd

```diff
@@ -0,0 +1,356 @@
+---
+title: "Numerical solution of SEIRD"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Numerical solution of SEIRD}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  markdown:
+    wrap: 72
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(comomodels)
+library(deSolve)
+library(ggplot2)
+library(tidyverse)
+library(glue)
+```
+
+## Introduction
+
+Compartmental epidemiological models (such as SEIR and its variants) are 
```

Comment:
Why SEIR but not SEIRD? Given the title of the Rmd is "Numerical solution of SEIRD".